### PR TITLE
fix: update links

### PR DESCRIPTION
**Pull request by joyqvq**
## Description 

Describe the changes or additions included in this PR.

## Test plan 

fix some links and make multisig page inter linked. 
---

## Release notes

Check each box that your changes affect. If none of the boxes relate to your changes, release notes aren't required.

For each box you select, include information after the relevant heading that describes the impact of your changes that a user might notice and any actions they must take to implement updates. 

- [ ] Protocol: 
- [ ] Nodes (Validators and Full nodes): 
- [ ] Indexer: 
- [ ] JSON-RPC: 
- [ ] GraphQL: 
- [ ] CLI: 
- [ ] Rust SDK:

### CODE REVIEW

File: docs/content/guides/developer/cryptography/multisig.mdx [hunk 1]
```diff
@@ -150,4 +152,5 @@ sui client execute-signed-tx --tx-bytes <TX_BYTES> --signatures <SERIALIZED-MULT
 
 ## Related links
 
-- [Multisig](../../../concepts/cryptography/transaction-auth/multisig.mdx): Definition of multisig transactions on the Sui network.
\ No newline at end of file
+- [Multisig](../../../concepts/cryptography/transaction-auth/multisig.mdx): Definition of multisig transactions on the Sui network and how to use it with CLI.
+- [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig address and sign transactions using the Sui SDK.
```
Comment (joyqvq) [inline docs/content/guides/developer/cryptography/multisig.mdx:156]: not sure how to link the sdk page's original page, feel free to fix @ronny-mysten

File: docs/content/guides/developer/cryptography/multisig.mdx [hunk 2]
```diff
@@ -3,7 +3,9 @@ title: Multisig Authentication
 description: Guide on how to create a multisig transaction and then submit it against a local network using the Sui CLI.
 ---
 
-The following steps demonstrate how to create a multisig transaction and then submit it against a local network using the [Sui CLI](/references/cli.mdx). A transaction can be the transfer of an object, the publish or upgrade of a package, the payment of SUI, and so on. To learn how to set up a local network, see [Connect to a Local Network](/guides/developer/getting-started/local-network.mdx).
+The following steps demonstrate how to create a multisig transaction and then submit it against a local network using the [Sui CLI](/references/cli.mdx). A transaction can be the transfer of an object, the publish or upgrade of a package, the payment of SUI, and so on. To learn how to set up a local network, see [Connect to a Local Network](/guides/developer/getting-started/local-network.mdx). 
+
+To learn more how to create multisig address and create multisig transaction from Typescript SDK, see [this page](https://sdk.mystenlabs.com/typescript/cryptography/multisig) for details.
```
Comment (ronny-mysten) [inline docs/content/guides/developer/cryptography/multisig.mdx:8]:
```suggestion
To learn more about how to create multisig addresses and create multisig transactions using the TypeScript SDK, see the [SDK documentation](https://sdk.mystenlabs.com/typescript/cryptography/multisig) for details. 
```

File: docs/content/guides/developer/cryptography/multisig.mdx (hunk 1 above)
Comment (ronny-mysten) [reply docs/content/guides/developer/cryptography/multisig.mdx:156]: My guess is you've never not figured something out correctly. This case included ;)

Comment (ronny-mysten) [inline docs/content/guides/developer/cryptography/multisig.mdx:156]:
```suggestion
- [Multisig Typescript SDK](https://sdk.mystenlabs.com/typescript/cryptography/multisig): How to create multisig addresses and sign transactions using the Sui SDK.
```

**Review by ronny-mysten — APPROVED** [review]
lgtm!

**Pull request merged**